modules/nodes-pods-autoscaling-custom-prometheus-config.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-prometheus-config_{context}"]
+= Configuring the custom metrics autoscaler to use Prometheus Cluster Monitoring
+
+You can use the installed Prometheus Cluster Monitoring as a source for metrics for the custom metric autoscaler. However, there is some additional configuration you must perform.
+
+You must perform the following tasks, as described in this section:
+
+* create a service account to get a token
+* create a Role
+* add that role to the service account
+* reference the token in the trigger authentication object used by Prometheus
+
+.Prerequisites
+
+* If you use a custom metrics autoscale for scaling on CPU/memory, your cluster administrator must have properly configured cluster metrics. You can use the `oc describe PodMetrics <pod-name>` command to determine if metrics are configured. If metrics are configured, the output appears similar to the following, with CPU and Memory displayed under Usage.
```

Comment:
This prerequisite here doesn't make much sense (it's only valid for CPU/Memory scaler), this is Prometheus one. Enabling OpenShift Monitoring for user workload should be here instead.